From fec31cf3f20aa97b7dc8b381e64f30eea4d4c2d2 Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Mon, 25 May 2020 16:11:51 +0200
Subject: [PATCH] tmpfiles: don't complain if we can't enable pstore in
 containers

(cherry picked from commit 203c07c95b91b4ae3b0c1ae3c92accdb71fd5e13)

Related: #2211416
---
 tmpfiles.d/systemd-pstore.conf | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tmpfiles.d/systemd-pstore.conf b/tmpfiles.d/systemd-pstore.conf
index cb600ec1ee..8b6a1dafc3 100644
--- a/tmpfiles.d/systemd-pstore.conf
+++ b/tmpfiles.d/systemd-pstore.conf
@@ -25,5 +25,5 @@
 # These changes are automatically applied on future re-boots.
 
 d /var/lib/systemd/pstore 0755 root root 14d
-#w /sys/module/printk/parameters/always_kmsg_dump - - - - Y
-w /sys/module/kernel/parameters/crash_kexec_post_notifiers - - - - Y
+#w- /sys/module/printk/parameters/always_kmsg_dump - - - - Y
+w- /sys/module/kernel/parameters/crash_kexec_post_notifiers - - - - Y
